python/lsst/ts/fiberspectrograph/dataManager.py

```diff
@@ -107,18 +108,26 @@ def make_fits_header(self, data):
         # as comments on each of these, but pydoc can't see them.
         hdr["FORMAT_V"] = FORMAT_VERSION
         hdr["OBSERVAT"] = "Vera C. Rubin Observatory"
-        hdr["INSTRUME"] = self.instrument
-        hdr["ORIGIN"] = self.origin
-        hdr["LOCATN"] = (None, "Location of Instrument")
+        hdr["INSTRUME"] = (self.instrument, "Type of Instrument.")
+        hdr["SERIAL"] = (self.serial, "Serial Number of Spectrograph.")
+        hdr["ORIGIN"] = (self.origin, "Name of the program that produced this data.")
```

Comment:
name of the program? I don't understand what you're referring to...

Reply:
I took that line from ts_electrometer:

> primary_hdu.header["ORIGIN"] = (
            self.csc.salinfo.name,
            "Name of the program that produced this data.",
        )

Reply:
ORIGIN in FITS means the producer. The FITS standard says:

```
KEYWORD:   ORIGIN
REFERENCE: [FITS Standard](http://archive.stsci.edu/fits/fits_standard/) 
STATUS:    reserved
HDU:       any
VALUE:     string
COMMENT:   organization responsible for the data
DEFINITION: The value field shall contain a character string
identifying the organization or institution responsible for creating
the FITS file.```

So in our case we should probably use "Vera C. Rubin Observatory" everywhere.

Reply:
In the Electrometer fits files, 'ORIGIN' shows `ts_electrometer`, i.e. the CSC name.

Reply:
Ok. That's not compliant with the FITS standard.